openedx/core/djangoapps/credentials/api/v1/views.py

```diff
@@ -1,3 +1,55 @@
 """
 Credentials API views (v1).
 """
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from django.http import HttpResponse, HttpResponseForbidden, HttpResponseBadRequest
+import json
```

Comment:
please move the json (standard library) import to a separate group, above this one.